stackit/internal/services/iaas/iaas_acc_test.go

```diff
@@ -22,10 +25,59 @@ import (
 	"github.com/stackitcloud/terraform-provider-stackit/stackit/internal/testutil"
 )
 
+//go:embed testdata/resource-security-group-min.tf
```

Comment:
personally I prefer to move multiple variable definitions in to a single block, like this:


```suggestion
var (
    //go:embed test1.txt
    test1 string

    //go:embed test2.txt
    test2 string
    ...

)
```

Reply:
done